### DISCUSSION TITLE
Help with my new i3 setup, suggestions? (i3-Gaps on Ubuntu)

### DISCUSSION THREAD

**Discussion by VOID1793** [i3/i3#5304] [Q&A] [answered]
I am fairly new to Linux, and VERY new to the whole "Linux as an aesthetic" thing and...I really like it! I want to be a part of it. Recently I downloaded what was, at the time, the latest version of Ubuntu (22.04 JJ) and messed with it, learned about Linux, filesystem, terminal, etc. Then I installed i3 as my first TWM! 

What follows is what I CURRENTLY have set up (Picture and Config file). My goal is to have the traditional Solarized-esque riced linux look (with all of the polish of Ubuntu/Debian behind the curtain). How can I improve my current configuration, given that i3-gaps is NOT supported on my distro yet? I would like to get as close as I can to a cool aesthetic/workflow look. 

Thank you!

![InkedScreenshot2](https://user-images.githubusercontent.com/119275782/204190721-f6f0b852-ace9-40d8-9119-d09f03c24f48.jpg)

[config.txt](https://github.com/i3/i3/files/10100333/config.txt)

**Comment by paulmyr** [comment]
I don't think that this is the right place to ask these kind of questions.
Maybe have a look [here](https://www.reddit.com/r/unixporn/) or [here](https://www.reddit.com/r/i3wm/).

I would recommend styling the bar to match your workflow and needs. I personally use polybar, but i3bar can look awesome too!

I think on ubuntu i3-gaps should still be available however; I also only have i3 installed, waiting for the 4.22 release.

Otherwise just keep customizing you colour schemes and fonts and your good to go.

**Reply by deanr72** [reply, marked as answer]
Why not give Regolith a try if you're intent on using a Ubuntu base?

https://regolith-desktop.com/
